docs/topics/commands.rst

```diff
@@ -430,6 +430,9 @@ Supported options:
 * ``--callback`` or ``-c``: spider method to use as callback for parsing the
   response
 
+* ``--meta`` or ``-m``: additional request meta that will be pass to the callback 
```

Comment:
Maybe this is a typo, should read "that will be **passed**"?

Reply:
Good catch @lopuhin! Yes, it was indeed a spelling mistake. Submitted a new commit to the PR that should fix that spelling mistake.